### ISSUE TITLE
parse language for 'northern' 'southern' etc. to make prayers sound more natural

### ISSUE THREAD

**Issue by squigglybob** [Pray4Movement/prayer-global-porch#119]

**Comment by ChrisChasm** [comment]
Yes. But we need to consider more localized languages. The people groups has language associated. And it it might be more accurate to the needs of the area. 

Suggestion: Go with available people group language, if not available go with country primary language.

**Comment by squigglybob** [comment]
@ChrisChasm should this one get pushed back to the 'new' column?

Did you mention something about the underlying data needing to change to match this? Or was it that the underlying data can't change and so we do need to massage it to change it's name when displaying it?

**Comment by ChrisChasm** [comment]
I think this is best handled at the database layer, instead of muddying up the code with conditionals.

**Comment by ChrisChasm** [comment]
I've added the grid_id_4770 column to the people_groups table, which will be a quick query to result languages geolocated to the 4770 locations. These results can be turned into new language concept blocks of prayer.
